prclz/reblock/planar_graph.py

```diff
@@ -690,8 +700,8 @@ def calc_steiner_tree(self,
 
         # Now get the MST of that complete graph of only terminal_vertices
         if "weight" not in H.es.attributes():
-            print("----H graph does not have weight, ERROR")
-            print("\t\t there are {}".format(len(terminal_vertices)))
+            error("----H graph does not have weight, ERROR")
+            error("\t\t There are %s", len(terminal_vertices))
```

Comment:
can we combine these two logging statements?

Reply:
done